toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -1380,6 +1398,9 @@ func addGroups(installChroot *safechroot.Chroot, groups []configuration.Group) (
 		err = installChroot.UnsafeRun(func() error {
 			return shell.ExecuteLive(squashErrors, "groupadd", args...)
 		})
+		if err != nil {
+			return fmt.Errorf("failed to add group: %w", err)
```

Comment:
nit: `"failed to add group:\n%w"`

Reply:
Please also say which group we failed to add. It's not guaranteed that name is going to be mentioned the output from `groupadd`.

Reply:
```suggestion
			return fmt.Errorf("failed to add group (%s):\n%w",  group.Name, err)
```